lib/sycamore/sycamore/transforms/similarity.py

```diff
@@ -156,6 +170,7 @@ def __call__(self, doc_batch: list[Document], query: str, score_property_name: s
     def score(self, inputs: list[tuple[str, str]]) -> list[float]:
         import torch
 
+        print(f"GPU: {torch.cuda.is_available()}")
```

Comment:
minor: Might want to remove this (or change to log).

Reply:
will do.